Register-And-Login-Screen/app/src/main/java/com/example/login_register_screen_ui/MainActivity.java

```diff
@@ -0,0 +1,47 @@
+package com.example.login_register_screen_ui;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.util.Pair;
+
+import android.app.ActivityOptions;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    Button login;
+    Button register;
+    ImageView imageView;
+    TextView textView;
```

Comment:
Class properties which are not going to be used outside the current class should be **private**

For example all UI related objects.